torch/distributed/checkpoint/state_dict.py

```diff
@@ -74,32 +74,34 @@ def gc_context():
 @dataclass
 class StateDictOptions:
     """
-    This specifies how get_state_dict/set_state_dict will work.
+    This dataclass specifies how get_state_dict/set_state_dict will work.
 
-    - ``fsdp_state_dict_type``: if the model is sharded by FSDP, what FSDP
-      state_dict type should be used.
-      The defalut value is SHARDED_STATE_DICT.
+    - ``fsdp_state_dict_type``: if the model contains FSDP sharded submodules,
+      what FSDP state_dict type should be used.
+      The default value is SHARDED_STATE_DICT.
 
     - ``ignore_frozen_params``: if the value is True, the returned state_dict
-      won't contain any parameters that are frozen -- the requires_grad is False.
+      won't contain any frozen parameters -- the ``requires_grad`` is False.
+      The default value is False.
+
+    - ``keep_submodule_prefixes``: when ``submodules`` is not None, this option
+      indicates whether to keep the submodule prefixes from the state_dict keys.
+      or example, if the submodule is ``module.pretrain`` and the full FQN of
+      the parameter is ``pretrain.layer1.weight`` of the param. When this option
+      is True, the parameter's key in the returned state_dict will be
+      ``layer1.weight``. If the options is False, the key will be
+      ``pretrain.layer1.weight``.
```

Comment:
Just for my understanding, keep_submodule_prefixes=True will return ``layer1.weight`` and keep_submodule_prefixes=False will return ``pretrain.layer1.weight``. I thought it was the other way around.